plugins/generator-1.15.2/forge-1.15.2/potion.definition.yaml

```diff
@@ -1,18 +0,0 @@
-templates:

```

Comment:
Keep potion for the ID as old workspaces will break. Also, in the current state, custom effects don't work as you didn't change the serialized name of the mod element.